pkgs/tools/admin/oci-cli/default.nix

```diff
@@ -22,6 +22,7 @@ let
         src = oldAttrs.src.override {
           inherit version;
           sha256 = "b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9";
+          hash = "";
```

Comment:
This is not ansible.

Reply:
Thanks, now I moved this hunk to the correct commit.